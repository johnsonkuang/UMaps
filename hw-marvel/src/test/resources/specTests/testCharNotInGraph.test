# Tests behavior when one or more nodes are not in the graph
LoadGraph graph singleNode.tsv

ListNodes graph

ListChildren graph 1

FindPath graph 1 2

FindPath graph 2 1

FindPath graph 2 3
