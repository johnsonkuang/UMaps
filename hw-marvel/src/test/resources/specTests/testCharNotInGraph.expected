# Tests behavior when one or more nodes are not in the graph
loaded graph graph

graph contains: 1

the children of 1 in graph are:

unknown character 2

unknown character 2

unknown character 2
unknown character 3